### PR TITLE
[SYCL] Fix USM/copy.cpp for double

### PR DESCRIPTION
**Pull request by aelovikov-intel**

### CODE REVIEW

**Comment by aelovikov-intel** [comment]
I don't have a local machine that can reproduce the failure, so will have to use pre-commit testing to see if it actually works.

File: SYCL/USM/copy.cpp [hunk 1]
```diff
@@ -71,6 +71,10 @@ template <typename T> void test(queue q, T val, T *src, T *dst, bool dev_dst) {
 }
 
 template <typename T> void runTests(queue q, T val, alloc kind1, alloc kind2) {
+  if ((std::is_same_v<T, double> || std::is_same_v<T, test_struct>)&&!q
```
Comment (romanovvlad) [inline SYCL/USM/copy.cpp:74]: Maybe we can just remove `double` from the `test_struct` or have two structs - with and without double.

Comment (AlexeySachkov) [reply SYCL/USM/copy.cpp:74]: I have applied this suggestion in #1242

**Comment by aelovikov-intel** [comment]
Closing in favor of https://github.com/intel/llvm-test-suite/pull/1242.

**Pull request closed without merging**